src/encoded/static/components/app.js

```diff
@@ -41,9 +41,9 @@ const portal = {
             title: 'Data',
             children: [
                 { id: 'functional-genomics', title: 'Functional Genomics data' },
-                { id: 'assaymatrix', title: 'Experiment matrix', url: '/matrix/?type=Experiment&status=released', tag: 'collection' },
+                { id: 'assaymatrix', title: 'Experiment matrix', url: '/matrix/?type=Experiment&status=released&replicates.library.biosample.perturbed=0', tag: 'collection' },
```

Comment:
@emro Looks like we’ll have a merge conflict here with ENCD-5267.

Reply:
I was partially through my review when you posted this and didn't see it, but it looks like we both had the same thought! One of us will just have to rebase after the other commit is merged.